mod/ckeditor/views/default/ckeditor/init.php

```diff
@@ -7,12 +7,14 @@
 
 ?>
 <script>
-// This global variable must be set before the editor script loading.
-CKEDITOR_BASEPATH = elgg.config.wwwroot + 'mod/ckeditor/vendors/ckeditor/';
+require(['elgg'], function (elgg) {
+	// This global variable must be set before the editor script loading.
```

Comment:
Just tested removing CKEDITOR_BASEPATH altogether, and it works. We have already defined it on the initial page load by extending js/elgg. IMHO, can safely remove, as we do not really need to redefine it.